test/coreSpec.js

```diff
@@ -441,8 +444,7 @@ describe('Core', function () {
         })
       })
     })
-
-    it('returns a quote when there is a direct (but remote) path', function * () {
+    xit('returns a quote when there is a direct (but remote) path', function * () {
```

Comment:
It's the same as `it.skip`, although I probably prefer `it.skip` since it's a bit more self-explanatory.